### PR TITLE
Initiate the is_training variable

### PR DESCRIPTION
**Pull request by DohanKim**
현재 BIMPM.py 파일의 dropout method는
```   
def dropout(self, v):
        return F.dropout(v, p=self.args.dropout, training=self.training)
```
위와 같이 self.training이라는 variable을 넘겨주고 있지만 이 variable은 어디서도 초기화되지 않습니다.
본래 training argument의 취지에 맞게 각 모드에 맞는 값을 초기화하고 args를 통해 넘겨주도록 했습니다.

### CODE REVIEW

**Comment by galsang** [comment]
제가 영어를 잘 하진 않지만 한국 분들 외의 다른 분들을 위해서 영어로 답변하겠습니다.

The self.training property is changed appropriately and automatically whenever you call model.train() and model.eval() methods in train.py and test.py, because the model is an instantiation of our BiMPM class which is a child of the **nn.module** class in PyTorch.

**Comment by DohanKim** [comment]
Thanks for the answer. I'll close the issue.
